os/dutils.sh

```diff
@@ -272,7 +286,9 @@ function TRAP_RUN()
 
 	echo "Executing: $TRAPCMD"
 	# execute TRAP script
-	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -it -w /root/tizenrt/tools/trap --privileged tizenrt/tizenrt:1.5.6 python3.7 $TRAPCMD
+	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -it -w /root/tizenrt/tools/trap --privileged tizenrt/tizenrt:1.5.6 python3.7 $TRAPCMD | tee $OUTPUTFILE
+
+	echo ">> Output is stored in $OUTPUTFILE"
```

Comment:
The output should not be a changed in git. Could you add prefix in the .gitignore?
To do this, the output file should have a prefix like `dutils_output_xxxxxx.txt`.
i.e. dutils_output_addr2line_<timestamp>.txt

Reply:
Updated